### ISSUE TITLE
Usage

### ISSUE THREAD

**Issue by test1230-lab** [varun-suresh/Clustering#10]
Could this work on a dir of unlabeled faces?

**Comment by marigoold** [comment]
No, you have to extract embeddings of those unlabeled faces using CNN or other extractors.
